spec/jwt/verify_spec.rb

```diff
@@ -87,14 +87,6 @@ module JWT
         Verify.verify_iat(payload, options)
       end
 
-      it 'must allow configured leeway' do
```

Comment:
It would be better to invert one of these tests to ensure that the leeway is ignored.